### PR TITLE
Change install to master, docs update

### PR DESCRIPTION
**Pull request by ahellander**
## Status

- [x] Ready
- [ ] Draft
- [ ] Hold

## Description
- Fixes to doc
- Change all default installs to master branch
- Updated release notes

## Types of changes

What types of changes does your code introduce to FEDn?

- [ ] Hotfix (fixing a critical bug in production)
- [ ] Bugfix
- [ ] New feature
- [x] Refactor, cleaning
- [x] Documentation update

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help! 
This is simply a reminder of what we are going to look for before merging your code._

- [x] This pull request is against **develop** branch (not applicable for hotfixes)
- [ ] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have read the [CONTRIBUTING](https://github.com/scaleoutsystems/fedn/blob/master/CONTRIBUTING.md) doc
- [ ] I have included tests to complement my changes
- [ ] I have updated the related documentation (if necessary) 
- [ ] I have added a reviewer for this pull request
- [ ] I have added myself as an author for this pull request

## Further comments

Anything else you think we should know before merging your code!

### CODE REVIEW

File: test/mnist-keras/extra-hosts.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+version: '3.3'
+services:
+ client:
+  extra_hosts:
+    combiner-name: 13.53.197.49
```
Comment (morganekmefjord) [inline test/mnist-keras/extra-hosts.yaml:5]: ?

File: test/mnist-pytorch/extra-hosts.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+version: '3.3'
+services:
+ client:
+  extra_hosts:
+    combiner-name: 13.53.197.49
```
Comment (morganekmefjord) [inline test/mnist-pytorch/extra-hosts.yaml:5]: ?

**Review by morganekmefjord — CHANGES_REQUESTED** [review]

File: test/mnist-keras/extra-hosts.yaml (hunk 1 above)
Comment (ahellander) [reply test/mnist-keras/extra-hosts.yaml:5]: What do you mean? Want a not-real IP instad in the example?

Comment (ahellander) [reply test/mnist-keras/extra-hosts.yaml:5]: ok, changed to non-funtional IPs in example.

**Pull request merged**